e2e/features/edit-cell-basic.feature

```diff
@@ -0,0 +1,7 @@
+Feature: Edit cell basic
+
+	Scenario: Save input chages when close cell editor
+        Given I am on "edit-cell-basic"
+        Then I open cell editor
```

Comment:
When I click "<column name>"[0]
Then editor value equals to "<something>"
When I change editor value to "<something new>"
And I click outside the editor
Then cell value "<column name>"[0] equals to "<something new>"